tools/tox.ini

```diff
@@ -15,7 +15,7 @@ changedir =
 
 commands =
   !flake8-!mypy: pytest --cov=tools --cov-report=term {posargs}
-  flake8: flake8 --append-config={toxinidir}/flake8.ini {posargs}
+  flake8: flake8 --append-config={toxinidir}/flake8.ini {toxinidir} {toxinidir}/../webdriver {posargs}
```

Comment:
I think I'd like to see a bit more consensus on https://github.com/web-platform-tests/rfcs/issues/171 prior to landing this. We don't often add new lint rules for tests, _and_ this is a different approach.

Currently, all the test lints are run via `./wpt lint`, and only tooling is linted here. This is especially notable because we only run the tools unittests when there are changes in `tools`, so we wouldn't even run this on most WebDriver PRs.

I also wouldn't expect this to be run in downstream repos, because by and large they're only running `wpt lint` and not the full tooling CI.

Reply:
@jgraham @foolip @DanielRyanSmith (as infra folks):

Any thoughts here?

Reply:
I also have another ask: In case there's no resolution of this in, say, 2-3 business days, then let's revert the `tox.ini` change and land the rest.

I think we can all agree that the lint fixes here are uncontroversial and valuable.

The contentious part seems to merely be how to enforce the linting in the infra.

Reply:
Yeah, landing the rest is totally non-objectionable, I think. The question is how we want to enforce it.